frontend/src/pages/CreateAccount.jsx

```diff
@@ -48,12 +48,35 @@ export default function CreateAccount() {
       inputErrors.push("Password and Confirm Password do not match");
     }
 
-    // If user input is invalid, set errors, otherwise clear errors and display success
+    // If user input is invalid, set errors
     if (inputErrors.length > 0) {
       setErrors(inputErrors);
+      return;
+    }
+    
+    setErrors([]);
+
+    // Send request to backend to create new account
+    const res = await fetch(`${import.meta.env.VITE_API_URL}/create-account`, {
```

Comment:
Example of how to use an .env variable to build the backend API URL dynamically. This will be useful when we get around to hosting the app later on. We can set an .env variable rather than having to manually change hardcoded URLs everywhere.